getting-started/kubernetes/k3s/quickstart.md

```diff
@@ -39,11 +39,44 @@ development.
    > CIDR by replacing 192.168.0.0/16 in the above command. 
    {: .alert .alert-danger}
 
+   K3s creates configuration files with `root` uid/gid, run the following command and change your user to `root`
```

Comment:
This sentence doesn't make sense. Two sentences?

 K3s creates configuration files with `root` uid/gid, run the following command and change your user to `root`

Reply:
Thanks for pointing it out. 👍